tictactoe.js

```diff
@@ -27,200 +27,207 @@
+let filled = false;
 
 /**
  * This is how you can query for elements. document.querySelector returns a single element. Note how "current-player" has a "." before it but on the element is defined like <div class="current-player">. The period before it
  tells javascript you are querying an element by it's classname. It is identical to how you write css
  */
- const currentPlayerIndicator = document.querySelector('.current-player')
- const scoreboard = document.querySelector('.scoreboard')
- /** Update the player indicator to show the current player. */
- currentPlayerIndicator.innerText = "Player " + (currentPlayer + 1)
-scoreboard.innerText = "SCORE \n\nPLAYER 1: " + playerOneWins + "\n\nPLAYER 2: " + playerTwoWins
+const currentPlayerIndicator = document.querySelector(".current-player");
+const scoreboard = document.querySelector(".scoreboard");
+/** Update the player indicator to show the current player. */
+currentPlayerIndicator.innerText = "Player " + (currentPlayer + 1);
+scoreboard.innerText =
+  "SCORE \n\nPLAYER 1: " + playerOneWins + "\n\nPLAYER 2: " + playerTwoWins;
 // You first step is to query for all of the element (hint use document.querySelectorAll)
-let gameBlocks = document.querySelectorAll(".tic-tac-toe__row__item")
+let gameBlocks = document.querySelectorAll(".tic-tac-toe__row__item");
 let clickedBlocks = document.querySelectorAll(".clicked");
-let emptyBlocks = document.querySelectorAll(".empty")
-let container  = document.querySelector(".container")
-
-for(let i=0; i <gameBlocks.length; i++){
-  gameBlocks[i].addEventListener("click", function(){
-    if(winner != true){
-      if(this.classList.contains("clicked")){
-        console.log("bad spot")
+let emptyBlocks = document.querySelectorAll(".empty");
+let container = document.querySelector(".container");
+
+for (let i = 0; i < gameBlocks.length; i++) {
+  gameBlocks[i].addEventListener("click", function() {
+    if (winner != true) {
+      if (this.classList.contains("clicked")) {
+        console.log("bad spot");
       } else {
-        this.innerText = currentPlayer ==  0 ? "X" : "O"
-        this.classList.add("clicked")
-        this.classList.remove("empty")
-        showWinner()
-        winner ? currentPlayer : currentPlayer == 0 ? currentPlayer++ : currentPlayer--
-        currentPlayerIndicator.innerText = (tieGame ? "CATS GAME" : ("Player " + (currentPlayer + 1) + (winner ? " WINS!" : "")))
-        scoreboard.innerText = "SCORE \n\nPLAYER 1: " + playerOneWins
-           + "\n\nPLAYER 2: " + playerTwoWins
+        this.innerText = currentPlayer == 0 ? "X" : "O";
+        this.classList.add("clicked");
+        this.classList.remove("empty");
+        showWinner();
+        winner
+          ? currentPlayer
+          : currentPlayer == 0
+          ? currentPlayer++
+          : currentPlayer--;
+        currentPlayerIndicator.innerText = tieGame
+          ? "CATS GAME"
+          : "Player " + (currentPlayer + 1) + (winner ? " WINS!" : "");
```

Comment:
String template literals are nice for situations like this (also a good use case for ternaries:

```

`Player ${currentPlayer + 1} ${winner ? 'WINS!' : ''}`
```